package.json

```diff
@@ -34,7 +34,8 @@
     "shelljs": "0.3.0",
     "chalk": "0.5.1",
     "grunt": "0.4.5",
-    "insight": "0.4.3"
+    "insight": "0.4.3",
+    "wiredep": "^1.0.0"
```

Comment:
Can you just put "1.0.0" ? We've had bad experiences with plugins upgrades, so now we fix the versions